lib/lightning_web/components/new_inputs.ex

```diff
@@ -412,7 +412,7 @@ defmodule LightningWeb.Components.NewInputs do
     ~H"""
     <p
       data-tag="error_message"
-      class="mt-3 inline-flex items-center gap-x-1.5 text-xs text-danger-600"
```

Comment:
I want to make lots of noise about this change!!!!

I don't really know what it'll break.

Reply:
It looks ok, it's the error description for a large set of inputs used in the app..

Before:
<img width="444" alt="Screenshot 2024-07-02 at 14 22 15" src="https://github.com/OpenFn/lightning/assets/144796/93555973-18c1-42cf-8daf-056d7864af45">
After:
<img width="444" alt="Screenshot 2024-07-02 at 14 22 00" src="https://github.com/OpenFn/lightning/assets/144796/316ba0c9-e554-484d-9302-00d51c956af8">


Thanks for flagging!